R/corpus-methods-quanteda.R

```diff
@@ -208,7 +208,11 @@ docnames <- function(x) {
 docnames.corpus <- function(x) {
     # didn't use accessor documents() because didn't want to pass
     # that large object
-    rownames(x$documents)
+    if (is.null(rownames(x$documents))) {
```

Comment:
Documents should have names always, but really old corpus object might not. This is a safeguard for errors in subsequent processes. If we were to give up backward compatibility, better to raise errors here.